app/services/induction/amend_participant_cohort.rb

```diff
@@ -100,12 +111,20 @@ def save
     def initialize(*)
       super
       @target_cohort_start_year = (@target_cohort_start_year || @schedule&.cohort_start_year).to_i
+      @reason_for_new_cohort ||= :unknown
     end
 
     def current_induction_record_updated?
       ActiveRecord::Base.transaction do
-        Induction::ChangeInductionRecord.call(induction_record:, changes: { induction_programme:, schedule: })
-        participant_profile.update!(school_cohort: target_school_cohort, schedule:)
+        Induction::ChangeInductionRecord.call(induction_record:,
+                                              changes: { induction_programme:,
+                                                         schedule:,
+                                                         training_status: :active })
```

Comment:
Im not sure about this, I think this field is mostly for provider actions and we should maintain? same below around the `training_status`